config/apis.ts

```diff
@@ -85,3 +115,5 @@ export const OKTA_AUTHZ_ENDPOINTS: Record<string, string> = oauthAPIList.reduce(
 export const OKTA_CONSUMER_APIS: string[] = Object.keys(OKTA_AUTHZ_ENDPOINTS);
 
 export const API_LIST: string[] = KONG_CONSUMER_APIS.concat(OKTA_CONSUMER_APIS);
+
+export const PRODUCTION_REQUEST_API_LIST: string[] = API_LIST.concat(INTERNAL_ONLY_APIS);
```

Comment:
Address Validation is going to be in here twice. it looks like the pattern Quokka was using was to put internal APIs in either `KONG_CONSUMER_APIS` or `OKTA_CONSUMER_APIS`, in which case `API_LIST` is still the source of truth. I'd say we should probably stick with that pattern (so as to avoid having multiple API lists for different purposes).

P.S. all internal only APIs are key auth except for Clinical Health, which is OAuth. (learned this yesterday, wanted to share for expedience)